src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Dumper;
+
+use Symfony\Component\Workflow\Definition;
+use Symfony\Component\Workflow\Marking;
+use Symfony\Component\Workflow\Transition;
+
+/**
+ * GraphvizDumper dumps a workflow as a graphviz file.
+ *
+ * You can convert the generated dot file with the dot utility (http://www.graphviz.org/):
+ *
+ *   dot -Tpng workflow.dot > workflow.png
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class GraphvizDumper implements DumperInterface
+{
+    private static $defaultOptions = array(
+        'graph' => array('ratio' => 'compress', 'rankdir' => 'LR'),
+        'node' => array('fontsize' => 9, 'fontname' => 'Arial', 'color' => '#333333', 'fillcolor' => 'lightblue', 'fixedsize' => true, 'width' => 1),
+        'edge' => array('fontsize' => 9, 'fontname' => 'Arial', 'color' => '#333333', 'arrowhead' => 'normal', 'arrowsize' => 0.5),
+    );
+
+    /**
+     * Dumps the workflow as a graphviz graph.
+     *
+     * Available options:
+     *
```

Comment:
1. It's done just bellow
2. Because theses options are relatives to **this** dumper.